### PR TITLE
Update to .NET 5.0 RC 1

### PR DESCRIPTION
**Pull request by doterik**
Thanks for the video and for sharing the code.
-
I had some initial problem, it may have been something with the SDK and 'RuntimeIdentifier'.
Changed that and upgraded to .NET 5.0 RC 1

Maybe others will have the same problem when they are inspired by your video.

### CODE REVIEW

**Comment by christoment** [comment]
Ah yes, I think people who had installed the latest Visual Studio Preview might get the RC1 automatically and will have issues running any existing .NET preview 8 project without modification.

Thanks for raising a pull request - I'll have a look 👍

File: BlazorWithRedux.csproj [hunk 1]
```diff
@@ -1,17 +1,17 @@
-<Project Sdk="Microsoft.NET.Sdk.Web">
+<Project Sdk="Microsoft.NET.Sdk.BlazorWebAssembly">
 
   <PropertyGroup>
     <TargetFramework>net5.0</TargetFramework>
-    <RuntimeIdentifier>browser-wasm</RuntimeIdentifier>
-    <UseBlazorWebAssembly>true</UseBlazorWebAssembly>
+    <!--<RuntimeIdentifier>browser-wasm</RuntimeIdentifier>-->
+    <!--<UseBlazorWebAssembly>true</UseBlazorWebAssembly>-->
```
Comment (christoment) [inline BlazorWithRedux.csproj]: We can definitely remove this 👍 

```suggestion
```

**Review by christoment — CHANGES_REQUESTED** [review]
We can remove the `RuntimeIdentifier` and `UseBlazorWebAssembly`, could you please remove those two lines?

Thanks for your help! 👍

**Comment by doterik** [comment]
Of course.
I now have removed the commented lines.

**Review by christoment — APPROVED** [review]
Looks good, thanks for the update!

**Pull request merged**